src/plugins/console/public/application/hooks/use_data_init/use_data_init.test.tsx

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
use_data_init.test.tsx -> use_data_init.test.ts

Reply:
Fixed. Thanks for the comment